rbac/internal/views.py

```diff
@@ -684,6 +684,88 @@ def populate_tenant_account_id(request):
     return HttpResponse('Invalid method, only "POST" is allowed.', status=405)
 
 
+def populate_tenant_org_id_view(request):
+    """View method for populating Tenant#org_id values by fetching from BOP.
+
+    POST /_private/api/utils/populate_tenant_org_id/
+
+    This endpoint:
+    1. Finds all tenants with empty org_id (excluding "public")
+    2. Fetches account_id to org_id mapping from BOP
+    3. Updates tenants with the org_id values
+    """
+    if request.method == "POST":
+        try:
+            # Find tenants with empty org_id
+            tenants_without_org_id = Tenant.objects.filter(org_id__isnull=True).exclude(tenant_name="public")
```

Comment:
Probably worth making this `select_for_update()` (and wrapping it the whole thing in a transaction)?